third-party/linshare/src/test/java/org/apache/james/linshare/client/LinshareAPITest.java

```diff
@@ -97,27 +101,27 @@ void listAllShouldReturnEmptyWhenNoUpload() {
 
     @Test
     void listAllShouldReturnAllUploadedDocuments() throws Exception {
-        Document firstDocument = user1LinshareAPI.uploadDocument(templateFile());
-        Document secondDocument = user1LinshareAPI.uploadDocument(templateFile());
+        Document firstDocument = uploadDocumentToUserSpace(USER1_MAIL, templateFile());
+        Document secondDocument = uploadDocumentToUserSpace(USER1_MAIL, templateFile());
 
         assertThat(user1LinshareAPI.listAllDocuments())
             .containsExactly(firstDocument, secondDocument);
```

Comment:
It's not related to this PR but i will take the opportunity to ask a question.
Do we care about the order of the elements returned by the API ? 
What the ordering rule is ?

if we don't care we can use `containsExactlyInAnyOrder`

Reply:
IMO: we upload directly one by one to userspace, so it will be ordered.
I think, we need to care about order of the elements returned by the API and also i can write some more test about it.

Reply:
> IMO: we upload directly one by one to userspace, so it will be ordered.

We should not assume order, we are just lucky here.

As far as I know LinShare spaces are unordered and sorting/limiting/filtering is done by Linshare fromtend.

Could you add a separate commit about decreasing our level of expectations here?

Reply:
> Could you add a separate commit about decreasing our level of expectations here?

What you mean?